hibernate-core/src/main/java/org/hibernate/cache/internal/EnabledCaching.java

```diff
@@ -428,7 +442,7 @@ public void evictDefaultQueryRegion() {
 
 	@Override
 	public void evictQueryRegion(String regionName) {
-		final QueryResultsCache cache = getQueryResultsCache( regionName );
+		final QueryResultsCache cache = namedQueryResultsCacheMap.get( regionName );
```

Comment:
```#getQueryResultsCache``` will create a new ```QueryResultsCache``` if one does not exist with the provided ```regionName```. 

I didn't think that it made sense to create a new ```QueryResultsCache``` when evicting, so I changed the line to lookup only an existing.

Should this be reverted?

Reply:
I think not creating the region on evict is better.  So I'd leave your change